cdn/dev/keyboard-search/install.js

```diff
@@ -71,6 +78,16 @@ function startAfterPageLoad_Windows(data) {
   }, 10);
 }
 
+function isLegacyFirefoxVersion() {
```

Comment:
What about passing in `browser` as a parameter since it's already assigned in l.43?

Reply:
yeah but then I still need to do it in the Linux call so figured it made no real difference...